packages/components/src/components/dropdown/items.tsx

```diff
@@ -1,30 +1,50 @@
-import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
+import classNames from 'classnames';
 import { getCurrencyDisplayCode } from '@deriv/shared';
 import Popover from '../popover';
 import Text from '../text';
+import { TListItem } from './utility';
+
+type TItem = {
+    className?: string;
+    handleSelect: (item: TListItem) => void;
+    has_symbol?: boolean;
+    onKeyPressed: (event: KeyboardEvent, item: TListItem) => void;
+    value?: string | number;
+    is_align_text_left?: boolean;
+    nodes?: any;
+    item: TListItem;
+};
 
-const Items = ({ items, ...props }) =>
-    items.map((item, idx) => {
-        return <Item key={idx} item={item} {...props} />;
-    });
+type TItems = Omit<TItem, 'item'> & {
+    items: TListItem[];
+};
 
-const Item = ({ onKeyPressed, value, item, handleSelect, nodes, has_symbol, is_align_text_left, className }) => {
-    const item_ref = React.useRef(null);
+const Items = ({ items, ...props }: TItems) => {
+    return (
+        <>
+            {items.map((item, idx) => {
+                return <Item key={idx} item={item} {...props} />;
+            })}
+        </>
+    );
+};
+
+const Item = ({ onKeyPressed, value, item, handleSelect, nodes, has_symbol, is_align_text_left, className }: TItem) => {
+    const item_ref = React.useRef<HTMLDivElement>(null);
     const symbol_type_class_name =
         item.text && typeof item.text === 'string' ? `symbols--${item.text.toLowerCase()}` : null;
 
     React.useEffect(() => {
         const removeListeners = () => {
             nodes.delete(item.value, item_ref.current);
```

Comment:
```suggestion
            nodes?.delete(item.value);
```
@niloofar-deriv Map delete() function accepts only 1 argument, should work without the 2nd one.